string file_name
configProp[] data

---------------
bool success
